saleor/graphql/app/schema.py

```diff
@@ -26,7 +26,7 @@ class Meta:
 
 
 class AppQueries(graphene.ObjectType):
-    ongoing_apps_installations = PrefetchingConnectionField(
+    ongoing_apps_installations = graphene.List(
         OngoingAppInstallation, description="List of all ongoing apps installations"
```

Comment:
@dominik-zeglen  Should it be `NonNull` list?

Reply:
I don't see any reason why there should be used null or list of nulls 🤷‍♂️